lib/context_adapter_communication.js

```diff
@@ -518,6 +534,22 @@ function getThirdPartyPayload(serviceDescriptor, operationDescriptor) {
 }
 
 /**
+ * Encodes the response sent to the Context Broker so no forbidden characters are used
+ *  The forbidden characters not allowed by the Context Broker are: < > " ' = ; ( )
+ * @param str The string to be encoded
+ * @return {string} The encoded string
+ */
+function encodeResponse(str) {
+  // encodeURIComponent() escapes all characters except the following: alphabetic, decimal digits, - _ . ! ~ * ' ( )
+  var encodedStr = encodeURIComponent(str);
```

Comment:
Fixed in https://github.com/telefonicaid/context-adapter/commit/cb846a6008d6351915189eb17dab241bdd0e4108